packages/account/src/account.test.ts

```diff
@@ -924,6 +924,29 @@ describe('Account', () => {
     });
   });
 
+  it('invalid request with getCoinsToSpend', async () => {
+    using launched = await setupTestProviderAndWallets({
+      walletsConfig: {
+        amountPerCoin: 100,
+        coinsPerAsset: 400,
+      },
+    });
+    const {
+      wallets: [wallet],
+      provider,
+    } = launched;
+
+    expect(provider.getChain().consensusParameters.txParameters.maxInputs.toNumber()).toBe(255);
+
+    await expectToThrowFuelError(
```

Comment:
Would not expect this to `throw`, just return a truncated response that would then throw when we attempted to execute a transfer that required `30_000`.